### PR TITLE
[BUG] Readme file needs Updation #4

### LINKED ISSUE #4: [BUG] Readme file needs Updation
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After PR #1 , some additional features were added into the project, such as Export to excel, which are not reflected back into the readme file.

### Expected Behavior

The Readme file needs to be up to date with latest features, Referring to the PR #1 , Update all the changes in the README file

### Steps To Reproduce

_No response_

### Anything else?

_No response_

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by jude-san**
This closes #4

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -8,15 +8,17 @@
...
 
-*   Add and delete invoice items
-*   Edit invoice item details
-*   Export invoice as a PDF
+- Add and delete invoice items
+- Edit invoice item details
+- Export invoice as a PDF
+- Data validation for PDF generation
+- Excel export
+- Date picker
```
Comment (guranshdeol) [inline Readme.md]: Line 20 and 21 are not required

**Review by guranshdeol — CHANGES_REQUESTED** [review]
Kindly delete the 2 rows

**Comment by jude-san** [comment]
> Kindly delete the 2 rows

done.

**Review by guranshdeol — APPROVED** [review]

**Comment by guranshdeol** [comment]
Thanks for Contributing!
Consider starring this repositry...

**Pull request merged**